print_designer/custom_fields.py

```diff
@@ -55,7 +55,6 @@
 			"fieldname": "print_designer_template_app",
 			"fieldtype": "Select",
 			"label": "Print Designer Template Location",
-			"default": "print_designer",
```

Comment:
This value will be set as a default by print designer, therefore kind of duplicate